pyproject.toml

```
@@ -0,0 +1,93 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "mkdocs"
+description = "Project documentation with Markdown."
+readme = "README.md"
+license = "BSD-2-Clause"
+authors = [
+    {name = "Tom Christie", email = "tom@tomchristie.com"},
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Environment :: Console",
+    "Environment :: Web Environment",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: Implementation :: CPython",
+    "Programming Language :: Python :: Implementation :: PyPy",
+    "Topic :: Documentation",
+    "Topic :: Text Processing",
+]
+dynamic = ["version"]
+requires-python = ">=3.7"
+dependencies = [
+    "click >=7.0",
+    "Jinja2 >=2.11.1",
+    "Markdown >=3.2.1, <3.4",
+    "PyYAML >=5.1",
+    "watchdog >=2.0",
+    "ghp-import >=1.0",
+    "pyyaml_env_tag >=0.1",
+    "importlib_metadata >=4.3; python_version < '3.10'",
+    "typing_extensions >=3.10; python_version < '3.8'",
+    "packaging >=20.5",
+    "mergedeep >=1.3.4",
+    "colorama >=0.4; platform_system == 'Windows'",
+]
+[project.optional-dependencies]
+i18n = [
+    "babel >=2.9.0",
+]
+
+[project.urls]
+Documentation = "https://www.mkdocs.org/"
+Source = "https://github.com/mkdocs/mkdocs"
+Issues = "https://github.com/mkdocs/mkdocs/issues"
+History = "https://www.mkdocs.org/about/release-notes/"
+
+[project.scripts]
+mkdocs = "mkdocs.__main__:cli"
+
+[project.entry-points."mkdocs.themes"]
+mkdocs = "mkdocs.themes.mkdocs"
+readthedocs = "mkdocs.themes.readthedocs"
+
+[project.entry-points."mkdocs.plugins"]
+search = "mkdocs.contrib.search:SearchPlugin"
+
+[tool.hatch.version]
+path = "mkdocs/__init__.py"
+
+[tool.hatch.build]
+artifacts = ["/mkdocs/**/*.mo"]
+[tool.hatch.build.targets.sdist]
+include = ["/mkdocs", "/LICENSE", "/pyproject.toml", "/README.md"]
```

https://hatch.pypa.io/latest/plugins/builder/sdist/#default-file-selection

```suggestion
include = ["/mkdocs"]
```